sparql/obsolete_replaced-by_terms_by_ontology.rq

```diff
@@ -0,0 +1,53 @@
+#+ summary: Find replaced terms by ontology
+#+ description: Choose an ontology and get back list of all obsolete terms with their terms replaced by it
+#+ pagination: 100
+#+ enumerate:
+#+   - ontology:
+#+    - <http://purl.obolibrary.org/obo/bspo.owl>
+#+    - <http://purl.obolibrary.org/obo/chebi.owl>
+#+    - <http://purl.obolibrary.org/obo/caro.owl>
+#+    - <http://purl.obolibrary.org/obo/iao.owl>
+#+    - <http://purl.obolibrary.org/obo/ro.owl>
+#+    - <http://purl.obolibrary.org/obo/emapa.owl>
+#+    - <http://purl.obolibrary.org/obo/bfo.owl>
+#+    - <http://purl.obolibrary.org/obo/ma.owl>
+#+    - <http://purl.obolibrary.org/obo/mi.owl>
+#+    - <http://purl.obolibrary.org/obo/mmo.owl>
+#+    - <http://purl.obolibrary.org/obo/envo.owl>
+#+    - <http://purl.obolibrary.org/obo/go.owl>
+#+    - <http://purl.obolibrary.org/obo/hp.owl>
+#+    - <http://purl.obolibrary.org/obo/ncbitaxon.owl>
+#+    - <http://purl.obolibrary.org/obo/mf.owl>
+#+    - <http://purl.obolibrary.org/obo/ncit.owl>
+#+    - <http://purl.obolibrary.org/obo/uberon.owl>
+#+    - <http://purl.obolibrary.org/obo/so.owl>
+#+    - <http://purl.obolibrary.org/obo/cl.owl>
+#+    - <http://purl.obolibrary.org/obo/fbbt.owl>
+#+    - <http://purl.obolibrary.org/obo/mp.owl>
+#+    - <http://purl.obolibrary.org/obo/bto.owl>
+#+    - <http://purl.obolibrary.org/obo/wbbt.owl>
+#+    - <http://purl.obolibrary.org/obo/zfa.owl>
+#+    - <http://purl.obolibrary.org/obo/xao.owl>
+#+    - <http://purl.obolibrary.org/obo/pr.owl>
+
+PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
+PREFIX owl: <http://www.w3.org/2002/07/owl#>
+PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
+PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
+PREFIX replaced_by: <http://purl.obolibrary.org/obo/IAO_0100001>
+SELECT ?obsolete_class ?replaced_by
+WHERE {
+  {
+    ?obsolete_class rdf:type owl:Class; 
+      owl:deprecated "true"^^xsd:boolean; 
+      replaced_by: ?replaced_by; 
+      rdfs:isDefinedBy ?_ontology_iri .
+  }
+  UNION
+  {
+    ?replaced_by rdf:type owl:Class;
+      oboinowl:hasAlternativeId ?obsolete_class .
+    ?obsolete_class rdfs:isDefinedBy ?_ontology_iri .
+  }
+  FILTER(isIRI(?replaced_by))
```

Comment:
just a question, is it possible to have a string or anything other than an IRI as range of replaced_by: ?

Reply:
I withdraw my question 😃

Reply:
😄